pom.xml

```diff
@@ -33,7 +33,8 @@
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-step-api</artifactId>
-    <version>2.10-SNAPSHOT</version>
+    <!-- Remove before merge, used to identify specific version for linked PRs, will cut as a beta -->
```

Comment:
Generally prefer that you `deploy` timestamped snapshots instead, so that the list of release versions in the repository is not permanently cluttered by experimental code (unless you are really intending to offer builds to users on the experimental UC).

Also BTW please file PRs from forks rather than from the origin repository.